gems/administrate/CVE-2016-3098.yml

```diff
@@ -0,0 +1,16 @@
+---
+gem: administrate
+cve: 2016-3098
+title: Cross-site request forgery (CSRF) vulnerability in administrate gem
+date: 2016-04-01
+
```

Comment:
Can you add a `url` of something like http://seclists.org/oss-sec/2016/q2/0 or some other public advisory? `url` is a required option.